src/mem/ruby/protocol/MOESI_AMD_Base-dir.sm

```diff
@@ -300,7 +304,11 @@ machine(MachineType:Directory, "AMD Baseline protocol")
         TBE tbe := TBEs.lookup(in_msg.LineAddress);
         CacheEntry entry := static_cast(CacheEntry, "pointer", L3CacheMemory.lookup(in_msg.LineAddress));
         if (in_msg.Type == DMARequestType:READ) {
-          trigger(Event:DmaRead, in_msg.LineAddress, entry, tbe);
+          if (L2isWB) {
+            trigger(Event:DmaReadWB, in_msg.LineAddress, entry, tbe);
+          } else {
+            trigger(Event:DmaRead, in_msg.LineAddress, entry, tbe);
+	      }
```

Comment:
tabs/spacing off on this line?

Reply:
I/Nagendra added an additional if() condition to check if L2 is WB. The additional spaces are from that, I think.

Reply:
I'm not sure how that can be for line 311?  But if it's an optical illusion, ok

Reply:
Surprisingly, I still see the right indentation in my github, vim views. But when I open the file in vim, I see that it uses a tab instead of spaces. Maybe a different tab/space configurations cause it to be formatted differently in different environments? I have now changed the tab to space. Hope that fixes it!